Last.FM/chain.js

```diff
@@ -0,0 +1,43 @@
+function TailUrl() { 
```

Comment:
just imagine you will have 1000 urls, will your approach work? + you mix all instances in one